packages/processing/setup.py

```diff
@@ -0,0 +1,87 @@
+"""pi-top Python SDK."""
+
+import os
+import sys
+
+from setuptools import setup
+
+if not sys.version_info >= (3, 7):
+    raise ValueError("This package requires Python 3.7 or above")
+
+HERE = os.path.abspath(os.path.dirname(__file__))
+
+__version__ = os.environ.get("PYTHON_PACKAGE_VERSION", "0.0.1.dev1")
+assert __version__ != ""
+
+__project__ = "pitop.processing"
+__author__ = "pi-top"
+__author_email__ = "deb-maintainers@pi-top.com"
+
+__url__ = "https://github.com/pi-top/pi-top-Python-SDK"
+__platforms__ = "ALL"
+
+# https://pypi.org/classifiers/
+__classifiers__ = [
+    "Development Status :: 5 - Production/Stable",
+    "Intended Audience :: Education",
+    "Intended Audience :: Developers",
+    "Topic :: Education",
+    "Topic :: System :: Hardware",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3",
+    "Programming Language :: Python :: 3.7",
+    "Programming Language :: Python :: 3.8",
+    "Programming Language :: Python :: Implementation :: PyPy",
+]
+
+__keywords__ = [
+    "pi-top",
+    "raspberrypi",
+    "battery",
+]
+
+__requires__ = [
+    f"pitop.common=={__version__}",
+    f"pitop.pma=={__version__}",
+    f"pitop.core=={__version__}",
+    "imutils>=0.5.4,<1.0.0",
+    "matplotlib>=3.3.4,<4.0",
+    "numpy>=1.19.5,<2.0",
+    "onnxruntime>=1.8.1,<2.0",
```

Comment:
This happens because pypi/testpypi don't provide a `onnxruntime` wheel for arm... that's why we compile and distribute it ourselves as a debian package. 
We should probably make that wheel available through a pypi-repository via PackageCloud and update any installation instruction to make sure users add it as an extra-index to their pip configuration.